contracts/amm.cairo

```diff
@@ -88,7 +99,7 @@ func do_trade{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
     strike_price: Math64x61_,
```

Comment:
@tensojka  do we want to switch away from Math64x61 here too?

Reply:
The migration would be a lot of work; we would have to migrate option tokens... And the impreciseness is not a problem with strike prices since they are nice round values.